command/genesis/polybft_params.go

```diff
@@ -155,6 +155,14 @@ func (p *genesisParams) generatePolyBftChainConfig(o command.OutputFormatter) er
 		return err
 	}
 
+	governorAdminAddr := types.ZeroAddress
+	if p.governorAdmin != "" {
+		// if no admin is defined, zero address will be the owner,
+		// meaning no new proposers and executors besides genesis validator
+		// set can be added later
```

Comment:
Only added or added+removed later? If they can be removed as well by governor admin, then I suppose update the comment.

Reply:
Done: https://github.com/0xPolygon/polygon-edge/pull/1728/commits/834a98adb036dc64e915da3b97cc18b4c190aa5a